client/ayon_deadline/plugins/publish/global/submit_publish_job.py

```diff
@@ -431,8 +431,8 @@ def process(self, instance):
         if audio_file and os.path.isfile(audio_file):
             publish_job.update({"audio": audio_file})
 
+        self.log.debug(f"Metadata json written to '{metadata_path}'")
```

Comment:
It's not written yet.
```suggestion
        self.log.debug(f"Writing metadata json to '{metadata_path}'")
```